src/ApiClient.php

```diff
@@ -54,6 +55,13 @@ public function __construct(HttpClient $httpClient, array $headers = [])
 
         $request = new Request('PUT', $uri, $headers, $content);
 
+        return $this->send($request);
+    }
+
+    private function send(RequestInterface $request) : PromiseInterface
+    {
+        $request = $request->withHeader('User-Agent', trim(sprintf('eLifeApiClient/%s %s', Version::get(), $request->getHeader('User-Agent')[0] ?? '')));
```

Comment:
Each of the clients. I've done it in two places to separate adding the `eLifeApiClient` details from the `GuzzleHttp` details.